Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM python:3.8
```

Comment:
Are we able to extend from the `python:3.8-alpine` image here? It's a bit more lightweight ([35MB](https://microbadger.com/images/python:3.8-alpine) vs [333MB](https://microbadger.com/images/python:3.8)). It's probably worth checking that the majority of our plugins work under alpine though (i.e. their native requirements are available).